src/react-components/home/HomePage.scss

```diff
@@ -24,7 +24,8 @@
   width: 250px;
 
   img {
-    width: 100%;
+    height: 83px;
+    width: 124px;
```

Comment:
Usually you want to avoid using "px" to define images because then you're hard-coding the aspect ratio for one single photo / it doesnt scale well for responsiveness.

Reply:
Yes, good call out. I didn't mean to leave this hard-coded